roles/filetree_create/templates/current_credentials.j2

```diff
@@ -3,19 +3,24 @@
 controller_credentials:
 {% endif %}
   - name: "{{ current_credentials_asset_value.name }}"
-    description: "{{ current_credentials_asset_value.description }}"
+    description: "{{ template_overrides_global.credential.description | default(current_credentials_asset_value.description) }}"
     credential_type: "{{ current_credentials_asset_value.summary_fields.credential_type.name }}"
-{% if current_credentials_asset_value.organization is defined and current_credentials_asset_value.organization is not none %}
+{% if template_overrides_global.credential.organization is defined or (current_credentials_asset_value.organization is defined and current_credentials_asset_value.organization is not none) %}
     organization: "{{ current_credentials_asset_value.summary_fields.organization.name }}"
 {% else %}
     organization: "{{ organization }}"
 {% endif %}
-{% if current_credentials_asset_value.inputs is defined and current_credentials_asset_value.inputs is not match('{}') %}
+{% if template_overrides_resources.credential[current_credentials_asset_value.name].inputs is defined
+      or template_overrides_global.credential.inputs is defined
```

Comment:
Makes it sense setting all the exported `credentials` with the same (`template_overrides_global`) inputs?

Reply:
I can think of multiple credentials of the same type... all with the same input values, being all the same credential... it's insane to generate redundant files when post-config will be needed anyway. In the best scenario, the redundant files should be deleted... I see no useful use case with this.